### PR TITLE
feat(RHICOMPL-1059): Add notice on external report deprecation

### PR DESCRIPTION
**Pull request by bastilian**
Screenshot 2020-11-06 at 14 45 53

### CODE REVIEW

File: src/SmartComponents/Reports/Reports.js [hunk 1]
```diff
@@ -63,7 +64,19 @@ LoadingView.propTypes = {
 
 const ReportsHeader = () => (
     <PageHeader>
-        <PageHeaderTitle title="Reports" />
+        <PageHeaderTitle title="Reports" className="pad-bottom" />
+        <Alert
+            variant="info"
+            isInline
+            title="Rule editing coming soon"
```
Comment (vkrizan) [inline src/SmartComponents/Reports/Reports.js:71]: This might be changed, e.g. to "Deprecation of external reports".

**Review by akofink — COMMENTED** [review]
Looks good, other than changing the title. :+1: Thanks!

File: src/SmartComponents/Reports/Reports.js (hunk 1 above)
Comment (bastilian) [reply src/SmartComponents/Reports/Reports.js:71]: 🤦

Comment (bastilian) [reply src/SmartComponents/Reports/Reports.js]: Corrected to what is in the mockup.

**Review by akofink — APPROVED** [review]
Thanks!

**Review by vkrizan — COMMENTED** [review]
The alert should be dismissable, as per mock. Once dismissed, I believe it should not be shown any more.

**Comment by bastilian** [comment]
@vkrizan The mockup seems to be incorrect. According to PM this message should not be dismissible.

**Pull request merged**